features/gba-dev/waardetabel-gba.feature

```diff
@@ -0,0 +1,247 @@
+
+      Gegeven landelijke tabel "Adellijke_Titel_Predicaat" heeft de volgende waarden
+      | code | omschrijving | soort     |
+      |   BS | barones      | titel     |
+      |   JH | jonkheer     | predicaat |
+      |    R | ridder       | titel     |
+
+
+  Rule: Bij een veld gedefinieerd als Waardetabel op een landelijke tabel wordt de omschrijving geleverd die in die tabel hoort bij de code
+
+    Abstract Scenario: Omschrijving bij code voor <tabel>
+      Gegeven de persoon met burgerservicenummer '000000176' heeft de volgende '<groepsnaam>' gegevens
+      | <lo3-naam>      |
+      | <waarde-code>   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000176                       |
+      | fields              | <propertynaam>  |
+      Dan heeft de response een persoon met de volgende '<propertynaam>' gegevens
+      | naam                | waarde               |
+      | <prefix>code                | <waarde-code>         |
+      | <prefix>omschrijving        | <waarde-omschrijving> |
+
+      Voorbeelden:
+      | tabel              | waarde-code  | waarde-omschrijving                   | groepsnaam              | lo3-naam                              | propertynaam                | prefix        |
+      | Gezagsverhouding   | 1D           | Ouder1 en een derde hebben het gezag  | gezagsverhouding        | indicatie gezag minderjarige (32.10)  | indicatieGezagMinderjarige  |               |
+      | Gemeenten          | 1326         | 's Heer Hendriks Kinderen             | verblijfplaats          | gemeente van inschrijving (09.10)     | gemeenteVanInschrijving     |               |
+      | Functie_adres      | B            | briefadres                            | verblijfplaats          | functieAdres.code (10.10)             | verblijfplaats              | functieAdres. |
+      | Verblijfstitel     | 09           | Art. 9 van de Vreemdelingenwet        | verblijfstitel          | aanduiding verblijfstitel (39.10)     | verblijfstitel              | aanduiding.|
+
+
+  Rule: voor adellijkeTitelPredicaat wordt ook soort geleverd die in die tabel hoort bij de code
+
+    Abstract Scenario: Omschrijving en soort bij code voor Adellijke titel/predicaat (02.20)
+      Gegeven de persoon met burgerservicenummer '000000188' heeft de volgende gegevens
+      | naam                                 | waarde    |
+      | adellijke titel of predicaat (02.20) | <waarde>  |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000188                       |
+      | fields              | naam.adellijkeTitelPredicaat    |
```

Comment:
N.m.m. illustreert dit scenario de bijbehorende rule onvoldoende. Doordat fields de waarde 'naam.adellijkeTitelPredicaat' wordt 'naam.adellijkeTitelPredicaat.soort' sowieso geleverd.
Zoals ik de rule interpreteer moet 'naam.adellijkeTitelPredicaat.soort' ook geleverd worden als fields de waarde 'naam.adellijkeTitelPredicaat.code' heeft. Geef fields die waarde of de waarde 'naam.adellijkeTitelPredicaat.code,naam.adellijkeTitelPredicaat.omschrijving'.

Reply:
Je interpreteert de rule niet zoals die bedoeld is. De uitzondering t.o.v. alle andere waardetabellen is dat ik soort ook geleverd wordt. Waar haal je anders de conclusie vandaan dat die sowieso geleverd wordt ? Die sowieso wordt dus hier beschreven. 
De conclusie dat soort ook geleverd moet worden als alleen code gevraagd wordt klopt niet. 
Ik heb de formulering van de Rule aangepast